test/dst/model.go

```diff
@@ -143,6 +143,8 @@ func (m *Model) Step(t int64, req *t_api.Request, res *t_api.Response, err error
 			return nil
 		case t_api.ErrAIOSubmissionQueueFull:
 			return nil
+		case t_api.ErrAIOStoreFailure:
+			return nil
```

Comment:
errors.Is here to differentaite from expected to unexpected. tolerated vs untolated.

Reply:
I also think we should do this cross-cutting where we inspect the error type to see if it an injected failure regardless of the code.